patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -337,18 +337,34 @@
 +
 +    /**
 +     * Determines if the specific ItemStack can be placed in the specified armor slot.
++     * @deprecated Override {@link #getEquipmentSlot(ItemStack)} to set the armor slot, and use
```

Comment:
[`@deprecated` tags go last in the Javadoc](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#orderoftags). Also, I know you didn't create this method, and you're deprecating it anyway, but can you fix up the rest of the Javadoc while you're at it?

[It should be:](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@param)

```
...
@param stack the {@code ItemStack} to check
@param armorType the armor slot to check. ID {@code 0}: helmet; {@code 1}: chest; {@code 2}: legs, {@code 3}: boots		 
@param entity the {@code Entity} trying to equip {@code stack} in the slot specified {@code armorType}
@return whether if {@code stack} can be inserted into the slot
...
```